packages/nodes-base/nodes/Onfleet/descriptions/WebhookDescription.ts

```diff
@@ -0,0 +1,131 @@
+import {
+	INodeProperties
+} from 'n8n-workflow';
+import { webhookMapping } from '../WebhookMapping';
+
+export const webhookOperations = [
```

Comment:
Why is this resource needed when the trigger is already handling these endpoints?

Reply:
I would remove it complely

Reply:
Well, the trigger node is a listener, however, we'd like to run webhook operations for other apps from n8n nodes.  i.e: 

- We want to be able to create an Onfleet webhook but n8n is not the listener
- We want to delete an Onfleet webhook however n8n is not the listener
- We want to list created Onfleet webhooks and they aren't necessary of n8n

In addition, this is part of the API offering and we'd like to keep them for uniformity.

Reply:
There are some use cases where n8n is used as middleware and not necessarily the webhook server. Hence we would like to keep the current setup so the webhook resource is handling endpoints outside of the trigger node.